ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/js/detailsofproduct.js

```diff
@@ -27,6 +27,15 @@ $(document).ready(function() {
     let productDetail;
     try {
      productDetail = JSON.parse(this.responseText);
+       let swfMailTo = document.getElementById('swfMailTo');
+ swfMailTo.href = "mailto:?"
```

Comment:
@uditnaagar777 plz remove js related to share component from pdp js and include the js in clientlibs of share component

Reply:
This is done. Thanks for the review